tests/FunctionalClientTest.php

```diff
@@ -308,15 +306,15 @@ public function testExecStreamStderrCustomEventWhileRunning($container)
     public function testExecStreamEmptyOutputWhileRunning($container)
     {
         $promise = $this->client->execCreate($container, array('true'));
-        $exec = Block\await($promise, Loop::get());
+        $exec = \React\Async\await($promise);
 
         $this->assertTrue(is_array($exec));
         $this->assertTrue(is_string($exec['Id']));
 
         $stream = $this->client->execStartStream($exec['Id'], true);
         $stream->on('end', $this->expectCallableOnce());
 
-        $output = Block\await(Stream\buffer($stream), Loop::get());
+        $output = \React\Async\await(Stream, Loop::get());
```

Comment:
Was getting rid of `\buffer($stream)` intentional? There's also a `Loop::get()` leftover.